backend/dissemination/models.py

```diff
@@ -439,15 +439,14 @@ class Passthrough(models.Model):
     We may not need this table. We can simply add three columns
     pertating to passthrough in FederalAward table
     """
+    award_seq_number = models.CharField(
```

Comment:
Let's rename award_seq_number to award_reference for consistency (just like report_id)